docs/en/seeding-classic-image.md

```diff
@@ -0,0 +1,314 @@
+---
+title: "Seeding a Classic Image"
+table_of_contents: true
+---
+
+# Seeding a Classic Image
+
+## Introduction
+A Classic image is seeded to include snaps and [assertions][assertions]. That
+is, you add the snaps and the assertions to the image so that on first boot it
+contains what your product needs. For example, a [gadget snap][gadget-snap] is
+normally seeded to point the system at a Serial Vault. Also, one seeds snaps
+to provide the software that customizes the system. This document explains
+what you may want to seed and how to do it.
+
+
+!!! Note:
+           Core images are seeded automatically with snaps and the model
+          assertion when created with the normal  ubuntu-image tool (available
+          as a snap in the beta channel). But, ubuntu-image does not yet
+          support building Classic images. However, the principles explained
+          here can be used to manually modify a Core image after it is built
+to
+          seed additional snaps and assertions.
+
+## What to seed
+Usually one needs to seed:
+
+ - A model assertion.
+ - An account assertion: for the brand SSO account used to authenticate the
+model assertion.
+ - An account-key assertion: to authenticate the key that signs the model
+assertion.
+ - A gadget snap and its assertions.
+
+!!! Note:
+          The gadget snap uploader must be the brand SSO account
+
+ - A core snap and its assertions.
+ - Any additional snaps (each with an assertion file).
+
+How to obtain these assertions and snap files is explained below.
+
+## Seed directory
+After you have built an image, you will need to add a seed directory to it
+(explained below): `/var/snapd/seed/`
+
+This directory has a certain structure. For example:
+
+ - `/var/snapd/seed/snaps/` contains the snap package files you want to seed.
+ - `/var/snapd/seed/assertions/` contains the assertions you want to seed.
+ - `/var/snapd/seed/seed.yaml` provides configuration data.
+
+All you need to do is add this directory to your image in the required place.
+On first boot, snapd handles the rest, verifying the snaps by their assertions
+and installing them, and verifying and importing any assertions (such as  the
+model assertion).
+
+### Creating your seed directory
+Create your seed directory as follows:
+
+1. Make the root seed directory:
+
+        $ mkdir seed
+
+1.  Make the assertions subdirectory
+
+        $ mkdir seed/assertions
+
+1.  Make the snaps subdirectory
+
+        $ mkdir seed/snaps
+
+The seed directory needs to be owned by root user.  Change ownership of these
+directories as follows:
+
+1. From the parent directory of seed/, change the owner recursively to root
```

Comment:
maybe `seed/`
also, perhaps better to say:
recursively change the ownership of all files in the seed directory to the root user
or something